source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,317 @@
+
+![Multidev PR workflow](/source/docs/assets/images/pr-workflow/multidev-git-pr-workflow.png)
+
+It is also common to set up automated tests to confirm that the project is working as expected; when tests are available, GitHub will run them and display the results of the tests with the pull request. Working on projects with comprehensive tests increases the development team's confidence that submitted pull requests will work correctly when they are integrated into the main build.
+
+When using GitHub and Composer to manage a Drupal site, only those files unique to the project are part of the project's main repository. Composer is used to fetch the external code needed by the project; a process running on Circle CI executes Composer, and ensures that the final composed build results are installed on Pantheon:
+
+![Artifact Deployment](/source/docs/assets/images/artifact-deployment.png)
+
+One advantage of managing code this way is that it keeps the change sets (differences) for pull requests as small as possible. If a pull request upgrades several external projects, only the external dependency metadata file will change; the actual code changes in the upgraded projects themselves are not shown.
+
+Generally, use of Composer is optional; however, some Drupal modules, such as the Address module require the use of Composer. If a site needs just one module that requires Composer, then it should manage all of its modules with Composer.
+
+## Before You Begin
+
+1.  To prepare your system for local development, install:
+
+    - [Composer](https://getcomposer.org).
+    - [Terminus](/docs/terminus/install/).
+    - The [Terminus Composer Plugin](https://github.com/pantheon-systems/terminus-composer-plugin#installation).
+    - The [Terminus Drupal Console Plugin](https://github.com/pantheon-systems/terminus-drupal-console-plugin#installation).
+    - The [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin#installation).
+
+<br>
+2.  Generate a [machine token](/docs/machine-tokens/) and [log in with Terminus](/docs/terminus/install/#authenticate).
+
+## Set Up a New Project
+
+Setting up multiple distributed services can be complicated, and, at the moment, this can only be done from the command line. Fortunately, the Terminus Build Tools Plugin makes this setup relatively simple.
+
+In this section we'll create a new Pantheon Site, a corresponding GitHub repository, and configure CircleCI to run tests.
+
+1.  To begin, create a new project:
+
+    ```bash
+    terminus build-env:create-project my-pantheon-project
+    ```
+Replace `my-pantheon-project` with the name of your new site.
+
+2.  The `create-project` command will prompt for any additional information it may need to set up the build workflow. The required information needed includes:
```

Comment:
@alexfornuto The items in `2` here are really part of step `1`; it's not really a second step. I think it would be better to just take out the `1` and `2` here, but I leave it up to you to decide.